pkgs/applications/networking/p2p/transgui/default.nix

```diff
@@ -64,10 +68,12 @@ stdenv.mkDerivation rec {
   '';
 
   meta = {
-    description = "A cross platform front-end for the Transmission Bit-Torrent client";
+    description =
+      "A cross platform front-end for the Transmission Bit-Torrent client";
```

Comment:
```suggestion
    description = "A cross platform front-end for the Transmission Bit-Torrent client";
```

Reply:
yeah I hate the formatting nixfmt had me use but I assumed it was a strict thing... I'll be sure to stop using and stick to my taste from now on :)

Reply:
Use nixpkgs-fmt which does not doe such changes.